### PR TITLE
Add WebUSB data.

### PR DESCRIPTION
**Pull request by jpmedley**
https://www.chromestatus.com/features/5651917954875392

### CODE REVIEW

File: api/USB.json [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "getDevices": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USB/getdevices",
```
Comment (Elchi3) [inline api/USB.json]: getDevices

File: api/USBAlternateInterface.json [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBAlternateInterface": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBAlternateInterface/USBAlternateInterface",
```
Comment (Elchi3) [inline api/USBAlternateInterface.json]: Please add a description clarifying that this is the constructor.
 `"description": "<code>USBAlternateInterface()</code> constructor",`

File: api/USBConfiguration.json [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBConfiguration": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBConfiguration/USBConfiguration",
```
Comment (Elchi3) [inline api/USBConfiguration.json]: Please add a description clarifying that this is the constructor.
 `"description": "<code>USBConfiguration()</code> constructor",`

File: api/USBEndpoint.json [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBEndpoint": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBEndpoint/USBEndpoint",
```
Comment (Elchi3) [inline api/USBEndpoint.json]: Please add a description clarifying that this is the constructor.
 `"description": "<code>USBEndpoint()</code> constructor",`

File: api/USBInTransferResult.json [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBInTransferResult": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBInTransferResult/USBInTransferResult",
```
Comment (Elchi3) [inline api/USBInTransferResult.json]: Please add a description clarifying that this is the constructor.
 `"description": "<code>USBInTransferResult()</code> constructor",`

File: api/USBInterface.json [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBInterface": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBInterface/USBInterface",
```
Comment (Elchi3) [inline api/USBInterface.json]: Please add a description clarifying that this is the constructor.
 `"description": "<code>USBInterface()</code> constructor",`

File: api/USBIsochronousInTransferPacket.json [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBIsochronousInTransferPacket": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBIsochronousInTransferPacket/USBIsochronousInTransferPacket",
```
Comment (Elchi3) [inline api/USBIsochronousInTransferPacket.json]: Please add a description clarifying that this is the constructor.
 `"description": "<code>USBIsochronousInTransferPacket()</code> constructor",`

File: api/USBIsochronousInTransferResult.json [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBIsochronousInTransferResult": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBIsochronousInTransferResult/USBIsochronousInTransferResult",
```
Comment (Elchi3) [inline api/USBIsochronousInTransferResult.json]: Please add a description clarifying that this is the constructor.
 `"description": "<code>USBIsochronousInTransferResult()</code> constructor",`

File: api/USBIsochronousOutTransferPacket.json [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBIsochronousOutTransferPacket": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBIsochronousOutTransferPacket/USBIsochronousOutTransferPacket",
```
Comment (Elchi3) [inline api/USBIsochronousOutTransferPacket.json]: Please add a description clarifying that this is the constructor.
 `"description": "<code>USBIsochronousOutTransferPacket()</code> constructor",`

File: api/USBIsochronousOutTransferResult [hunk 10]
```diff
@@ -0,0 +1,148 @@
+{
```
Comment (Elchi3) [inline api/USBIsochronousOutTransferResult]: This needs to be a .json file

File: api/USBIsochronousOutTransferResult [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBIsochronousOutTransferResult": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBIsochronousOutTransferResult/USBIsochronousOutTransferResult",
```
Comment (Elchi3) [inline api/USBIsochronousOutTransferResult]: Please add a description clarifying that this is the constructor.
 `"description": "<code>USBIsochronousOutTransferResult()</code> constructor",`

File: api/USBOutTransferResult.json [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "USBOutTransferResult": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/USBOutTransferResult/USBOutTransferResult",
```
Comment (Elchi3) [inline api/USBOutTransferResult.json]: Please add a description clarifying that this is the constructor.
 `"description": "<code>USBOutTransferResult()</code> constructor",`

**Comment by jpmedley** [comment]
PTAL

By the way, I used this: http://jsoneditoronline.org/#/

**Comment by Elchi3** [comment]
Thanks, yeah, I'm using that site, too.

api/USBIsochronousOutTransferResult is still not a JSON file, otherwise LGTM.

**Comment by jpmedley** [comment]
Sorry about that.

**Comment by Elchi3** [comment]
(Will be packaged and deployed on Thursday)

**Pull request merged**